content/en/real_user_monitoring/session_replay/browser/troubleshooting.md

```diff
@@ -26,6 +26,7 @@ If you experience unexpected behavior with Datadog Session Replay, use this page
 Session Replay does not support:
 
 - The following HTML elements: `iframe`, `video`, `audio`, or `canvas`
+  - To render iFrames in Session Replay, you can separately instrument the iFrame code. For iFrames that span across sub-domains, use `trackSessionAcrossSubdomains: true`. After correct instrumentation, iFrames and their parent windows appear as distinct entities within the same session, embedding iFrame replays directly into their parent windows is not supported.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ ‏It is unclear what we mean by  "distinct entities". We could be more specific.